skills/README.md

```diff
@@ -0,0 +1,109 @@
+# Skills
+
+## Skills Object
+
+```
+{
+    "name": <name of skill>,
+    "on": <when skill was added>,

```

Comment:
createdOn/ createdAt would be better right?
Are we not thinking about levels here?

Reply:
1. yes descriptive name would be better & i had mentioned on & by as `awarded_on` & `awarded_by` but ankush had suggested to just keep it short

Reply:
2. i think not. it was not mentioned by sumit in the referred ticket

Reply:
createdAt is an standard term used in many projects that I am aware of.....
I am curious on why @ankushdharkar agreed with on and by.
However, if we have come to conclusion after discussion. We are cool